camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/ConfigInheritanceYamlTest.java

```diff
@@ -72,6 +72,7 @@ public void setUp() throws Exception {
         
         emptyFile = Files.createTempFile("ConfigInheritanceYamlTest", ".txt");
         emptyFile2 = Files.createTempFile("ConfigInheritanceYamlTest2", ".txt");
+        emptyFile3 = Files.createTempFile("ConfigInheritanceYamlTest3", ".txt");
```

Comment:
I'd still prefer deleting immediately in tearDown, as `Os.newTempFile` only sets `deleteOnExit()` (so will not delete it if the process is killed).